.sequelizerc

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
```

Comment:
This file is used for the sequelize-cli package.  It is just a default file that sequelize-cli looks for when running commands on the command line.  If this file is not present and a sequelize-cli command is run then it automatically places files and folders in the main directory.

So, for example if I run:
  --sequelize model:generate --name <schema table name> --attributes ...

this command will look for the .sequelizerc file and if it is found it will generate a new model and a new migration file and place them in the server/migration folder and server/model folder.

Without the file running the command would just place both new files in new directories in the main LUCCA directory.